about/builds.md

```diff
@@ -6,6 +6,6 @@ data: { dateless: "true" }
 permalink: /about/builds.html
 ---
 
-While there are several [simple ways for developers to experiment and explore with WPE]({{'/about/exploring.html' | url}}) generally, for real, shipping projects or good performance testing embedded systems are typically custom builds.  To make this easier, there is also [meta-webkit](https://github.com/Igalia/meta-webkit) which provides build recipes  WebKit based runtimes and browsers for use with OpenEmbedded and/or Yocto.
+While there are several [simple ways for developers to experiment and explore with WPE]({{'/about/exploring.html' | url}}) generally, shipping projects or good performance-testing embedded systems are typically custom builds.  To make this easier, there is also [meta-webkit](https://github.com/Igalia/meta-webkit), which provides build recipes, WebKit based runtimes, and browsers for use with OpenEmbedded and/or Yocto.
```

Comment:
I think the original is based on something I wrote and even that doesn't make sense to me on this one.  Perhaps it should say "shipping products for embedded systems are typically performance tuned custom builds" or something?

Reply:
Okay, so with a tiny edit, the text would be:

> While there are several simple ways for developers to experiment and explore with WPE, generally shipping products for embedded systems are performance-tuned custom builds.

Does that make sense in context?

Reply:
Yeah. Basically the idea is these make it very easy to explore WPE, but they aren't remotely indicative (currently, at least) of the actual performance you can get with WPE.  This isn't the sort of thing people who don't work on embedded would normally think of and ... well, we're working on maybe making it less so, but at least currently if you run the full embedded chromium on the same build it seems radically faster on a number of things even where it might not normally be in a product.

Reply:
I think extra clarity is needed here, as I didn’t catch the emphasis.  New proposal:

> While there are several simple ways for developers to experiment and explore with WPE, none are tuned for performance.  Generally, shipping products for embedded systems are performance-tuned custom builds.

But does that link with the following sentence about meta-webkit?

Reply:
Yes, I think so... you don't have to fully "solve" building your own too - there are "easy" (for people with more skills than me and work on embedded systems) ways to do it.  I am happy to merge all of the rest of the things in here but I sort of wonder if @aperezdc or @philn could  review/comment on this short para to keep me honest/accurate

Reply:
> I think extra clarity is needed here, as I didn’t catch the emphasis. New proposal:
> 
> > While there are several simple ways for developers to experiment and explore with WPE, none are tuned for performance.  Generally, shipping products for embedded systems are performance-tuned custom builds.
> 
> But does that link with the following sentence about meta-webkit?

Yeah I think it does :) The performance tuning doesn't happen so much in meta-webkit itself, but in the BSP package it is included in, where compiler flags can be tuned. In general meta-webkit should already fulfil basic optimisations, at least for the products we worked on so far. Maybe Adrian has something to add though.